pkg/datapath/linux/node.go

```diff
@@ -704,7 +704,7 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 		oldKey = oldNode.EncryptionKey
 	}
 
-	if n.nodeConfig.EnableIPSec && !n.subnetEncryption() {
+	if n.nodeConfig.EnableIPSec && !n.subnetEncryption() && !n.nodeConfig.EnableIPSec {
```

Comment:
Am I reading this correctly? We check if the node has IPSec, and then check if it **doesn't** have IPSec? Maybe this was a typo?

Reply:
its a typo ;) thanks. I forgot to commit the fix.

Reply:
```
if n.nodeConfig.EnableIPSec && !n.subnetEncryption() && !n.nodeConfig.EncryptNode {
```
is the intended line.